stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/viewmodel/messages/MessageListViewModel.kt

```diff
@@ -50,11 +50,11 @@ import io.getstream.chat.android.livedata.utils.Event as EventWrapper
  * @param globalState Global state of OfflinePlugin. Contains information
  * such as the current user, connection state, unread counts etc.
  */
-public class MessageListViewModel @JvmOverloads constructor(
+public class MessageListViewModel constructor(
```

Comment:
we can also remove the `constructor` keyword given that we no longer have annotations